src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -570,4 +575,28 @@ void QMCDriverNew::checkLogAndGL(Crowd& crowd, const std::string_view location)
     throw std::runtime_error(std::string("checkLogAndGL failed at ") + std::string(location) + std::string("\n"));
 }
 
+void QMCDriverNew::measureImbalance(const std::string& tag) const
+{
+  ScopedTimer local_timer(timers_.imbalance_timer);
+  Timer only_this_barrier;
+  myComm->barrier();
+  std::vector<double> my_barrier_time(1, only_this_barrier.elapsed());
+  std::vector<double> barrier_time_all_ranks(myComm->size(), 0.0);
+  myComm->gather(my_barrier_time, barrier_time_all_ranks, 0);
+  if (!myComm->rank())
+  {
+    auto const count  = static_cast<double>(barrier_time_all_ranks.size());
+    const auto max_it = std::max_element(barrier_time_all_ranks.begin(), barrier_time_all_ranks.end());
+    const auto min_it = std::min_element(barrier_time_all_ranks.begin(), barrier_time_all_ranks.end());
+    app_log() << std::endl
```

Comment:
Why so many std::endl?  I see the semantics of that as << '\n', and flush the stream.  Certainly there is no need to flush this output more than once.

Reply:
It should not matter much. Clang-format doesn't work well with '\n'.